pom.xml

```diff
@@ -2,19 +2,18 @@
   <parent>
     <groupId>org.jenkins-ci</groupId>
     <artifactId>jenkins</artifactId>
-    <version>1.54</version>
+    <version>1.60</version>
   </parent>
 
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.jenkins-ci</groupId>
   <artifactId>executable-war</artifactId>
   <name>Executable War 'header'</name>
   <version>1.46-SNAPSHOT</version>
   <description>Defines the main() method for a self-contained web application that kickstarts
     embedded Winstone and runs a web application without needing a servlet container.</description>
 
   <properties>
-    <java.level>6</java.level>
+    <java.level>8</java.level>
```

Comment:
It's been ~3.5 years since Jenkins weeklies worked on Java 7, and more than five years since Java 6 was supported. This seems like a reasonable minimum now.

Haven't looked closely enough at this PR to approve, but I'm fine with this change.

Reply:
Fine with me roo